server/sql/sell-everywhere-script.sql

```diff
@@ -0,0 +1,353 @@
+--PostgresSQL script
+
+--for uuid_generated_v4
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+--PRODUCT
+CREATE TABLE IF NOT EXISTS product
+(
+    id uuid DEFAULT uuid_generate_v4(),
+    name VARCHAR(100) NOT NULL,
+    description VARCHAR(5000),
+    price DECIMAL(7,2) DEFAULT 0,
```

Comment:
please never store things related to money in precision point types. Use int to store coins and denominations if needed